src/app/clusters/camera-av-stream-management-server/camera-av-stream-management-server.cpp

```diff
@@ -1657,6 +1721,13 @@ void CameraAVStreamMgmtServer::HandleAudioStreamAllocate(HandlerContext & ctx,
         ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Status::ConstraintError);
     });
 
+    bool streamUsageSupported = std::find_if(mRankedVideoStreamPriorities.begin(), mRankedVideoStreamPriorities.end(),
```

Comment:
Again: this comes after constraint checks per spec.

Reply:
The current Spec actually has the streamUsageSupported check at the beginning, before the constraint errors.

Reply:
The constraint errors in the spec are in the IM, before you ever get to the cluster.